### ISSUE TITLE
Issue with imports

### ISSUE THREAD

**Issue by victoriafrench** [threadheap/serverless-ide-vscode#54]
The following causes the IDE to report the file as invalid with `[Serverless IDE] Incorrect type. Expected "object".`

```yaml
functions:
  - ${file(hello-world/functions.yml)}
  - ${file(simple-test/functions.yml)}
```

Is this syntax been added to the IDE yet?

**Comment by pavelvlasov** [comment]
Hey @victoriafrench! Thanks for reporting 👍 

This syntax is not supported just yet.
If you don't mind to contribute, you can change the type around https://github.com/threadheap/serverless-ide-vscode/blob/master/packages/serverless-framework-schema/json/aws/functions/functions.json to support array of functions.

Otherwise, I'll try to figure out that this week.

**Comment by pavelvlasov** [comment]
Resolved in v0.5.6
